testing/checkin/runner/testing.rs

```diff
@@ -10,3 +14,23 @@ pub struct Output {
 pub struct TestFunctions {
   pub functions: Vec<(String, v8::Global<v8::Function>)>,
 }
+
+#[derive(Default)]
+pub struct TestData {
+  pub data: HashMap<(String, TypeId), Box<dyn Any>>,
+}
```

Comment:
Nitpick: I think you could use `GothamState` from `deno_core` directly?

Reply:
I don't think so as this has a name/typeid pair, while `GothamState` is just typeid.